content/guides/use-case/jupyter/share.md

```diff
@@ -0,0 +1,143 @@
+---
+description: Learn how to share your JupyterLab environment and notebooks in a Docker image.
+keywords: jupyter, notebook, python, jupyterlab, data science
+title: Share your JupyterLab environment and notebooks
+toc_min: 1
+toc_max: 2
+---
+
+## Overview
+
+This section guides you on incorporating your Jupyter notebooks into a Docker
+image and sharing it with the broader data science community through Docker Hub.
+
+By adding your notebooks to an image, you create a portable and replicable
+research environment that can be easily accessed and used by other data
+scientists. This process not only facilitates collaboration but also ensures
+that your work is preserved in an environment where it can be run without
+compatibility issues.
+
+You'll learn how to include your notebooks in a Docker image, how to publish
+this image on Docker Hub, and how you or others can download and run your image.
+This enables others to download, use, and build upon your work, fostering open
+science and collaborative development.
+
+## Prerequisites
+
+- You have installed the latest version of [Docker Desktop](../../../get-docker.md).
+- You have completed [Customize your JupyterLab environment](customize.md) and
+  saved a notebook to your local machine.
+
+## Add a notebook to a Docker image
```

Comment:
discussion: Doesn't this seem a bit like an anti-pattern? We're baking project files into an image.

If we used a volume for the workbook, we could distribute the image and the workbook separately, but both via Docker Hub. We can push the volume as an OCI artifact: https://docs.docker.com/docker-hub/oci-artifacts/#push-a-volume

What do you think?

Reply:
I think the idea is that this ties the notebook and environment together. Otherwise, someone can run the data in a different image and possibly have different results. I'm not super sold on the idea, so let's change it.